StyleGuide.md

```diff
@@ -489,6 +490,14 @@ Links are marked with inline text surrounded by square brackets followed by the
     [--max-offset](#flags-max-offset)
     ```
 
+#### GitHub issues and pull requests
+
+[Release notes](https://www.cockroachlabs.com/docs/releases/index.html) and [technical advisories](https://www.cockroachlabs.com/docs/advisories/index.html) contain links to individual GitHub issues and pull requests.
+
+Issues and pull requests should be referenced by their corresponding number, prepended with `#`.
```

Comment:
Really minor nit: You could switch up this sentence to make it active: 

You should reference issues and pull requests by their corresponding number, prepended with `#`.

OR 

Reference issues and pull requests by their corresponding number, prepended with `#`.